encoding/protobuf/header.go

```diff
@@ -0,0 +1,36 @@
+package protobuf
+
+import (
+	"encoding/base64"
+
+	"github.com/gogo/protobuf/proto"
+	spb "google.golang.org/genproto/googleapis/rpc/status"
+	"google.golang.org/grpc/status"
+)
+
+const _grpcStatusDetailsHeaderKey = "grpc-status-details-bin"
```

Comment:
Can we share this constant with transport/grpc/outbound.go somehow? Is there a common place for this?